wrapper_writer/parsers.py

```diff
@@ -114,6 +114,11 @@ def regex_parser(self, data):
                 doc_string = re.sub("\n?@.*\n?", "", doc_string).strip()
             elif match.group(2) is not None:
                 # If the second group is not none then this a function match
+                # This gets the function access modifier, protected def, def, private def
+                access = match.group(2).replace("def", "").strip()
+                if access == "":
```

Comment:
Sometimes pycharm will highlight warning "expression can be simplified"
`if not access`